projects/noloan2/app/src/main/java/hasadna/noloan2/MainActivity.java

```diff
@@ -74,4 +85,52 @@ public void onClick(View v) {
         writeUser(user);
     }
   }
+  
```

Comment:
General comment - since the app can't really work without reading SMS, maybe we should ask for the permissions when the app opens, and if not granted the permissions, close the app?
That way we can later assume we have the permissions.